app/models/letter_template.rb

```diff
@@ -0,0 +1,13 @@
+class LetterTemplate < ApplicationRecord
+  validates :name, :body, :template_type, presence: true
+
+  enum template_type: {
+    dispatch: 'dispatch',
```

Comment:
`despatch` !!

Reply:
Nope :) - they're both valid UK English but Alex and the team chose to settle on `dispatch` e.g.       `ready_to_dispatch`. We still use `de-` in places accidentally, but it's correct here

![image](https://user-images.githubusercontent.com/1161161/65603579-d650f480-df9d-11e9-88ae-d93e03beae0e.png)

Reply:
Booo. 51st state here we come